### DISCUSSION TITLE
[Monolog] Production won't work

### DISCUSSION THREAD

**Discussion by grzegorz-stolarz** [symfony/symfony#45287] [Q&A]
> We use monolog-bundle in version ^3.7 (exactly 3.7.1)

`bin/console c:c --env=prod -vvv`

Results are:

`[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]                                     
  Unrecognized option "monolog" under "monolog". Available options are "channels", "handlers", "use_microseconds".`

I've used default production `monolog.yaml`

More details on monolog issue tracker - [https://github.com/symfony/monolog-bundle/issues/427](url)

**Comment by alexisLefebvre** [comment]
The link is not clickable, here it is: https://github.com/symfony/monolog-bundle/issues/427

Please put the error message in this discussion. Other users will appreciate having all the information in one place. Put ``` on new lines before and after your code to format it.
